account_cutoff_base_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © 2018 Magnus (Willem Hulshof <w.hulshof@magnus.nl>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Cut-off Base Operating Unit',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'OU adaptation for Base module Account Cut-offs',
+    'author': 'Magnus,Odoo Community Association (OCA)',
+    'website': 'http://www.magnus.nl',
```

Comment:
@whulshof URL Should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)